app/views/users/_comments_section.html.erb

```diff
@@ -1,46 +1,65 @@
-        <div class="breadcrumb"><a href="<%= @user.path %>">View All Activity</a></div>
-        <header>
-          <% if @user.comments_count > 250 %>
-            <h4>Last 250 Comments</h4>
-          <% else %>
-            <h4>All <%= @user.comments_count %> Comments</h4>
-          <% end %>
-        </header>
+        <div class="crayons-card__header">
+          <h4 class="crayons-card__headline">
+            <% if @user.comments_count > 250 %>
+              Last 250 comments
+            <% else %>
+              All <%= @user.comments_count %> comments
+            <% end %>
+          </h4>
+          <div class="crayons-card__actions">
+            <a href="<%= @user.path %>" class="crayons-link--branded fs-base">
+              View all activity
+            </a>
+          </div>
+        </div>
       <% else %>
-        <header>
-          <h4>Recent Comments</h4>
-        </header>
+        <div class="crayons-card__header">
+          <h4 class="crayons-card__headline">
+            Recent comments
+          </h4>
+        </div>
       <% end %>
     <% end %>
     <% @comments.each do |comment| %>
       <% if comment.commentable.present? && comment.commentable.published && !comment.deleted %>
-        <a href="<%= comment.path %>">
-          <div class="single-comment <%= "strong-comment" if comment.ancestry.nil? %>">
-            <span class="comment-title">re: <%= comment.commentable.title %></span>
-            <span class="comment-date">
+        <a href="<%= comment.path %>" class="profile-comment-row ">
+          <span class="fw-bold fs-base">
+            <%= comment.commentable.title %>
+          </span>
+          <div class="inline-flex items-center gap-2">
+            <div class="fs-s comment-preview">
+              <div class="crayons-avatar crayons-avatar--m mr-1">
+                <img alt="<%= @user.username %>%" src="<%= @user.profile_image_90 %>" class="crayons-avatar__image" />
+              </div>
+              <span class="fw-medium mr-1">
+                <%= @user.name %>
+              </span>
+              <span class="color-base-80">
+                <%= truncate(strip_tags(comment.processed_html), length: 64).html_safe %>
+              <span>
+            </div>
+            <span class="color-base-60 fs-s comment-date whitespace-nowrap">
               <time datetime="<%= comment.decorate.published_timestamp %>"><%= comment.readable_publish_date %></time>
             </span>
-            <div class="comment-preview"><%= truncate(strip_tags(comment.processed_html), length: 64).html_safe %></div>
           </div>
         </a>
       <% end %>
     <% end %>
     <% if params[:view] != "comments" && @user.comments_count > 250 %>
-      <div class="breadcrumb"><a href="<%= @user.path %>/comments">Last 250 Comments</a></div>
+      <div class="pt-3 pl-4 pb-3 pr-4">
+        <a href="<%= @user.path %>/comments" class="fs-base">
+          View last 250 Comments
```

Comment:
This isn't related to your work and isn't a blocker, but I think we should probably extract out the `250` and the `8` in this file (right now it is hardcoded, but located in different places, which will make it annoying to change and keep track of 😓). I'll open a separate issue for it, but just noting it here for other reviewers :)

Reply:
@vaidehijoshi great suggestion, and thank you for creating that issue 💯